### PR TITLE
Tendermint integration

### PR DESCRIPTION
**Pull request by pienkowb**

### CODE REVIEW

File: manifest-schema.graphql [hunk 1]
```diff
@@ -8,7 +8,8 @@ type Schema {
```
Comment (otaviopace) [inline manifest-schema.graphql:8]: You'll have to rebase this with `master`, @maoueh just refactored this

File: src/protocols/index.js [hunk 2]
```diff
@@ -64,6 +67,9 @@ module.exports = class Protocol {
         'near-mainnet',
         'near-testnet'
       ],
+      tendermint: [
+        'cosmoshub-4'
```
Comment (otaviopace) [inline src/protocols/index.js:71]: I'm curious what this means, ELI5 👶

File: src/protocols/tendermint/contract.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module.exports = class TendermintContract {
+  static identifierName() {
+    return 'address'
```
Comment (otaviopace) [inline src/protocols/tendermint/contract.js:3]: Hmmm perhaps we should change this, from my understanding we'll have "chains" instead of "contracts".

File: src/protocols/tendermint/contract.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  }
+
+  validate() {
+    return {
+      valid: true,
+      error: "",
```
Comment (otaviopace) [inline src/protocols/tendermint/contract.js:9]: This should be `null` when validation goes right.

File: src/protocols/tendermint/contract.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+module.exports = class TendermintContract {
```
Comment (otaviopace) [inline src/protocols/tendermint/contract.js:1]: I think you can delete this file 🙂 

It's only required if you call `validateContractValues` in the `validation` step, but it's empty in `tendermint`'s `subgraph.js` file (`immutable.List()`).

**Comment by otaviopace** [comment]
I'm actually kind of surprised you had to do so little code to integrate with `tendermint`, it feels like something is missing (did I do the right abstractions here so it's easy? 😅 ).

Well, when chatting with @azf20 we were talking if it would be possible to fetch the "chain" types for a subgraph just like we do the `ABI` for EVM compat subgraphs.

So then you could send that on deployment to `graph-node` and they could be used. 🙂 🎉 

(we can discuss this in person)

**Comment by roignpar** [comment]
I tried rebasing with latest master, but it turned out to be a pain, so I manually redid (and adapted) all the changes on top of latest master.

File: src/protocols/tendermint/contract.js (hunk 3 above)
Comment (otaviopace) [inline src/protocols/tendermint/contract.js:3]: We should fix this later, since Tendermint doesn't have `addresses`.

**Review by otaviopace — APPROVED** [review]

**Comment by otaviopace** [comment]
I'll make the CI run just to be sure then we can merge this 😊
